tiatoolbox/models/architecture/idars.py

```diff
@@ -58,20 +60,20 @@ class IDaRS(CNNModel):
 
     """
 
-    def __init__(self, backbone, num_classes=1):
+    def __init__(self, backbone, num_classes=1) -> None:
+        """Initializes :class:`IDaRS`."""
         super().__init__(backbone, num_classes=num_classes)
 
     @staticmethod
-    # noqa: E800
     def preproc(image: np.ndarray):
-        """Define preprocessing steps.
+        """Defines preprocessing steps.
```

Comment:
I think it should be 'Define' not 'Defines'. Generally, function descriptions should be in the imperative case e.g. 'Compute', 'Solve...' examples at: https://docs.scipy.org/doc/scipy/reference/linalg.html.

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/commit/c2e1cd4ccfd6aaeb32664f23d4f6634cddb90090